dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -282,4 +286,22 @@ public static void checkToolList(String log) {
         Assert.assertTrue(log.contains("DESCRIPTION"));
         Assert.assertTrue(log.contains("Git Repo"));
     }
+
+    public static void restartElasticsearch() throws Exception {
+        final DockerClient docker = DefaultDockerClient.fromEnv().build();
+        List<Container> containers = docker.listContainers();
+        Optional<Container> elasticsearch = containers.stream().filter(container -> container.image().contains("elasticsearch"))
+                .findFirst();
+        if (elasticsearch.isPresent()) {
+            Container container = elasticsearch.get();
+            try {
+                docker.restartContainer(container.id());
+                // Wait 15 seconds for elasticsearch to become ready
+                // TODO: Replace with better wait
+                Thread.sleep(15000);
```

Comment:
I guess you can't work around it, but how much time does this add to the tests?

Reply:
should be about a min or so, we don't have a lot of tests